From e3f9298588d3ca546a4b5d6bbcd7f5b79c06dfdb Mon Sep 17 00:00:00 2001
From: Shawn Betts <sabetts@gmail.com>
Date: Sun, 12 Jul 2009 18:23:25 -0700
Subject: check for getline in configure.in

Bug-Debian: 552894
Applied-Upstream: 4ad0b38fb53506d613c4b4f7268dadfcedae9b8e

This fixes a build error encountered on glibc 2.10 systems
---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 68b6fa9..6654cb7 100644
--- a/configure.in
+++ b/configure.in
@@ -155,7 +155,7 @@ AC_CHECK_HEADERS(unistd.h stdarg.h)
 dnl Checks for typedefs, structures, and compiler characteristics.
 
 dnl Checks for library functions.
-AC_CHECK_FUNCS(getopt getopt_long setsid setpgid setpgrp putenv vsnprintf usleep)
+AC_CHECK_FUNCS(getopt getopt_long setsid setpgid setpgrp putenv vsnprintf usleep getline)
 
 AC_TYPE_SIGNAL
 
